src/components/notification/index.tsx

```diff
@@ -46,23 +46,28 @@ export const Notification: React.FC<NotificationProps> = ({
   };
 
   const primaryButtonDefaultStyle: CSSProperties = {
+    maxWidth: 'calc(50% - 32px)',
```

Comment:
and here

Reply:
We've added a margin for the button, so using a percentage might not function correctly in this context.